release/README.md

```diff
@@ -1,27 +1,27 @@
 ## Intel Data Center GPU Driver Container Images For OpenShift Release
 ### General
-Intel data center GPU driver container images are certified and released on the [Red Hat Ecosystem catalog](https://catalog.redhat.com/).To make the driver container images managed by KMM operator and used by people easily, below name patten is used for naming and tracking each image released.
+Intel data center GPU driver container images are certified and released on the [Red Hat Ecosystem catalog](https://catalog.redhat.com/). To make the driver container images managed by KMM operator easy to use and track, the following naming convention is adopted.
 
-* **RepositoryName:Tag** image naming patten
+* **RepositoryName:Tag** image naming pattern
   * **RepositoryName:** intel-data-center-gpu-driver-container
   * **Tag:** DriverVersion--KernelVersion
     * **DriverVersion:** X.Y.Z
-      * **X:** update for driver architecture change or major features 
+      * **X:** update for driver architecture or major feature change
         
-        **Example:** 0: drivers for RHEL8.x  1: drivers for RHEL9.x
-      * **Y:** updates for driver bug fixing or minor features support
+        **Example:** 0 = drivers for RHEL8.x,  1 = drivers for RHEL9.x
```

Comment:
should we continue use : instead of = since : is used in many other places here?

Reply:
Updated. :)